crates/turbopack-ecmascript/tests/tree-shaker/analyzer/complex/output.dot

```diff
@@ -0,0 +1,19 @@
+digraph {
+    0 [ label = "\"getDog#1\"" ]
+    1 [ label = "\"dog#1\"" ]
+    2 [ label = "\"setDog#1\"" ]
+    3 [ label = "\"newDog#2\"" ]
```

Comment:
newDog is in this graph, but actually we are not interested in tracking that, are we? We only care about top level declarations and statements.

Reply:
I was cautious because of parameters with a default value, but I think it does not matter.

```js

let cat = "cat"

function foo(p = cat) {
    
}

```

`foo => cat (initial)` would be enough.

I'll remove them from the graph.